transaction/std/src/transaction_builder.rs

```diff
@@ -162,6 +162,16 @@ impl<FPR: FogPubkeyResolver> TransactionBuilder<FPR> {
             .push(InputMaterials::Signable(input_credentials));
     }
 
+    /// Add a View Only Input to the transaction.
+    ///
+    /// # Arguments
+    /// * `input_view_only_materials` - Materials required to construct the
+    ///   onetime private key for the input.
```

Comment:
Except they're not actually enough to construct the onetime private key, no? Or maybe I got confused...

Reply:
Ah, yeah that's right, in addition to the spend private key! Will note that